extensions/GOOGLE/SPV_GOOGLE_user_type.asciidoc

```diff
@@ -45,13 +46,13 @@ extension.
 
 Overview
 --------
-This extension provides one new decoration, UserTypeGOOGLE, that allows HLL
+This extension provides one new decoration, UserTypeGOOGLE, that allows HLSL
```

Comment:
I suspect this is actually supposed to be HLL for "High level languages"  (This isn't only useful for HLSL is it?)

Reply:
Makes sense. I think it's only used for HLSL right now (the resource types below are HLSL specific), but it wouldn't have to be limited to that.